extra-modules/extra-phishing/src/main/kotlin/com/kotlindiscord/kord/extensions/modules/extra/phishing/PhishingExtension.kt

```diff
@@ -278,19 +282,24 @@ class PhishingExtension(private val settings: ExtPhishingBuilder) : Extension()
         val domains: MutableSet<String> = mutableSetOf()
 
         for (match in settings.urlRegex.findAll(content)) {
-            val found = match.groups[1]!!.value.trim('/')
+            val found = match.groups[1]?.value?.trim('/')
+
+            if (found == null) {
+                continue
+            }
+
             var domain = found
 
             if ("/" in domain) {
                 domain = domain
                     .split("/", limit = 2)
-                    .first()
```

Comment:
I don't think it's possible for this to be missing (or if it is, the user supplied a custom regex and an error should be thrown)